src/_includes/partials/template-list.njk

```diff
@@ -0,0 +1,27 @@
+{% set templates = [cardlist.type, locale | upper] | join %}
+{% set templateCards = collections.all | eleventyNavigation(templates) %}
+
+{% if cardlist.type === 'pageTemplates' and cardlist.state === 'published' %}
+  <div class="my-500">
```

Comment:
`my-500` needs to be changed to 600 please to match the new spacing classes.

Reply:
```suggestion
  <div class="my-600">
```

Reply:
Done!